docs/container_registry.md

```diff
@@ -42,15 +42,16 @@ lifetime like this:
 ```bash
 mkdir certs
 cd certs
-openssl req -new -newkey rsa:4096 > registry.csr
-openssl rsa -in privkey.pem -out registry.key
+# Generate a random password password_file used in the next commands
+openssl rand -hex -out password_file 32
```

Comment:
Why do we need a password file to generate the key with a password ?

Reply:
Because otherwise, `openssl` asks for a "passphrase" interactively. So it can't be automatized.

That's just to be consistent with the script `generate-registry-certificates.sh` that does it all automatically when the env var `GITLAB_REGISTRY_GENERATE_INTERNAL_CERTIFICATES` is set to `'true'`.

With the commands as they are, they can be copy-pasted directly, without adding any additional input (no passphrase).